manifest.json

```diff
@@ -21,11 +18,12 @@
   ],
   "content_scripts": [
     {
-      "matches": ["*://www.linkedin.com/jobs/*"],
-      "css": ["css/global.css"]
+      "matches": ["*://www.linkedin.com/*"],
+      "css": ["css/global.css"],
+      "run_at": "document_start"
```

Comment:
The CSS will be loaded now when the page begins to load. The default value here if unspecified is `document_idle`.